src/d3a/models/strategy/__init__.py

```diff
@@ -635,10 +644,15 @@ def can_settlement_offer_be_posted(
             offer_energy, offer_price, unsettled_energy_kWh, market,
             replace_existing=replace_existing)
 
+    @property
+    def spot_market(self) -> "Market":
+        """Return the spot_market member of the area."""
+        return self.area.spot_market
```

Comment:
Nit: I favour the idea of adding this property. It would be great if we could adapt the rest of the codebase (replacing the old `area.spot_market` occurrences with the new property).

Reply:
I'll merge it now and open a separate PR for this and the other comment. Heads up though, I will disable pylint for this